src/triage/component/postmodeling/contrast/utils/test_conn.py

```diff
@@ -0,0 +1,2 @@
+from sqlalchemy import create_engine
+db_engine = create_engine('postgresql://food_user:ivanymariacomenenWok30@0.0.0.0:5434/food')
```

Comment:
Should this really be in the repo?

Reply:
OK. Is this not something that triage can be made to do using its built-in configuration?

Reply:
Yes, it's only for testing purposes. Triage has its own `engine`, but since I'm trying to test things, I create my own `engine` to the `dirty_duck` db.

Reply:
Sure. And that's a different engine than what triage creates? A different connection? (Not sure if in your test you could just use triage's.) Regardless, sounds like you can uncommit it? :smile:

Reply:
Please use an ebvironment variable